documentation/tut/toy-example-from-scratch.rst

```diff
@@ -3,23 +3,14 @@
 Toy example: Scheduling a battery, from scratch
 ===============================================
 
-Let's walk through an example from scratch! We'll ... 
-
-- install FlexMeasures
-- create an account with a battery asset
-- load hourly prices
-- optimize a 12h-schedule for a battery that is half full
+Let's walk through an example from scratch! We'll optimize a 12h-schedule for a battery that is half full.
 
 What do you need? Your own computer, with one of two situations: either you have `Docker <https://www.docker.com/>`_ or your computer supports Python 3.8+, pip and PostgresDB. The former might be easier, see the installation step below. But you choose.
 
 Below are the ``flexmeasures`` CLI commands we'll run, and which we'll explain step by step. There are some other crucial steps for installation and setup, so this becomes a complete example from scratch, but this is the meat:
```

Comment:
Likewise, we don't need the latter part ("There are some othere ...") - it lives in the setup already.